app/forms/openid_connect_token_form.rb

```diff
@@ -112,7 +112,7 @@ def validate_code
 
   def validate_code_verifier
     expected_code_challenge = remove_base64_padding(identity.try(:code_challenge))
-    given_code_challenge = remove_base64_padding(Digest::SHA256.base64digest(code_verifier.to_s))
+    given_code_challenge = Digest::SHA256.urlsafe_base64digest(code_verifier.to_s)
     return if expected_code_challenge == given_code_challenge
```

Comment:
no need to block the deploy, but spotting this here made me realize this would be a good spot to use secure_compare, so I made a separate PR: https://github.com/18F/identity-idp/pull/6226

Reply:
Pulled that change in!